style.css

```diff
@@ -13,170 +13,479 @@
+  border: 1px solid #fff;
+  min-height: 140px;
+  padding: 10px;
+  margin: 10px 0;
+  transition: padding 0.3s ease-in-out;
+}
+
+.form_text_area:hover {
+  padding-left: 20px;
+}
+
+.submit {
+  float: left;
+}
+
+@media screen and (min-width: 768px) {
+  .header {
+    padding: 0 50px;
   }
 
-  .nav-item-action {
-    font-size: 15px;
-    font-weight: 500;
-    color: #344563;
-    line-height: 20px;
-    padding: 10px 12px;
-    text-decoration: none;
+  .header-manu {
+    display: none;
   }
 
-  .nav-item-action:hover {
-    color: #fff;
-    background-color: #344563 !important;
+  .nav-items {
+    display: flex;
+    align-self: center;
+    justify-self: center;
+    flex-direction: row;
   }
 
   .mobile-nav-items {
-    top: 0;
-    left: 0;
-    bottom: 0;
-    width: 100%;
-    height: 100vh;
     display: none;
   }
 
   .article {
+    background-image: url("./resources/images/header-shapes\ Deasktop\ BG.png");
     display: flex;
-    min-height: 100vh;
+    min-height: 113vh;
     flex-direction: column;
-    background-color: #fff;
-    border-bottom-left-radius: 70px;
+    background-color: white;
+    border-bottom-left-radius: 15%;
     background-repeat: no-repeat;
-    background-size: contain;
-    background-position: center;
-    background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+    background-size: 100%;
+    background-position: 99% 25%;
   }
 
   .article-item1 {
-    margin: 20% 12% 15%;
+    align-self: center;
+    max-width: 920px;
   }
 
   .headline-title {
     color: #172b4d;
     font-weight: 900;
     font-size: 40px;
-    font-style: bolder;
     line-height: 52px;
     display: flex;
     flex: none;
-    margin-bottom: 20px;
     font-family: Poppins-bold, sans-serif;
   }
 
+  .headline-container {
+    align-self: center;
+    justify-self: center;
+    max-width: 100vw;
+  }
+
   .headline-content {
-    font-size: 16px;
+    font-size: 20px;
     color: #344563;
     font-style: normal;
     font-weight: 400;
-    line-height: 30px;
-    flex: none;
-    margin-left: 0;
-    margin-right: 24px;
-  }
-
-  .headline-container {
-    display: grid;
-    row-gap: 25px;
-  }
-
-  .social-links {
-    display: flex;
-    list-style: none;
-    margin-top: 10px;
-    padding-left: 1%;
-    padding-right: 10px;
+    line-height: 35px;
+    margin-right: 25%;
   }
 
   .social-title {
-    font-size: 16px;
+    font-size: 20px;
     font-weight: 600;
     text-decoration: none;
     color: #7f8cff;
-    line-height: 50px;
-    width: 120px;
-    height: 24px;
+    line-height: 60px;
   }
 
   .work-section {
-    margin: 80px 0;
-    display: grid;
-    row-gap: 80px;
-    padding: 10px;
-    justify-items: center;
+    margin: 100px;
   }
 
   .work-card {
-    background: #fff;
-    border-radius: 10px;
-    min-height: 3em;
-    border: 1px solid #c6c6c6;
-    padding: 20px 15px;
-    display: grid;
+    display: flex;
+    flex-direction: row;
   }
```

Comment:
- [ ] Why don't you use `grid` instead of `flex` to display the content of your cards in a row direction? I think using `grid` is going to be much better so kindly consider using it so that both your card image and text will have equal fractions of the main container.
`HINT:` You can give it a `display: grid` and `grid-template-columns: 1fr 1fr`. This way both children of the card container will have equal fractions and will also help make your website even more responsive. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1170" alt="Screenshot 2023-06-14 at 20 32 54" src="https://github.com/Uzair-Manzoor/Portfolio/assets/107192076/b8041807-ee6e-47e3-8254-057581263a4c">
</td>
<td>
<img width="1030" alt="Screenshot 2023-06-14 at 20 31 37" src="https://github.com/Uzair-Manzoor/Portfolio/assets/107192076/7f354649-e35f-46de-849b-a4d54041a78f">
</td>
</tr>
</table>